### DISCUSSION TITLE
`parserOptions.sourceType` not mentioned in docs

### DISCUSSION THREAD

**Discussion by silverwind** [typescript-eslint/typescript-eslint#9182] [Community Feedback]
The [current parser docs](https://typescript-eslint.io/packages/parser/) do not mention the `parserOptions.sourceType` option at all, but I think it was supported at some point in time.

Is it safe to remove it from configs? [This comment](https://github.com/typescript-eslint/typescript-eslint/issues/8518#issuecomment-1954971255) make me believe it is, but I would like a confirmation.

**Comment by bradzacher** [comment]
https://typescript-eslint.io/contributing/discussions

As per our contributing guide for discussions. We do not use discussions for questions or support. Please use one of the listed alternatives.

**Comment by silverwind** [comment]
I think this qualifies as a technical discussion. I see the [option is recognized](https://github.com/typescript-eslint/typescript-eslint/blob/38ccbe3c7144b08d5af998d49fe65778829ec699/packages/types/src/parser-options.ts#L75), but absent in the docs, so it's a docs bug.

**Reply by bradzacher** [reply]
> I think this qualifies as a technical discussion

But you used the "Community Feedback" category?

---

Either way as per our contributing guide
https://typescript-eslint.io/contributing/discussions

> Technical Discussions: Deeper questions about typescript-eslint's architecture and/or APIs

Asking about an option isn't really a "deeper question".
